pkg/plugins/backendplugin/backend_plugin.go

```diff
@@ -197,9 +197,13 @@ func (p *BackendPlugin) checkHealth(ctx context.Context) (*pluginv2.CheckHealth_
 	return res, nil
 }
 
-func (p *BackendPlugin) callResource(ctx context.Context, req CallResourceRequest) (*CallResourceResult, error) {
+func (p *BackendPlugin) callResource(ctx context.Context, req CallResourceRequest) (CallResourceResultStream, error) {
 	p.logger.Debug("Calling resource", "path", req.Path, "method", req.Method)
 
+	if p.core == nil || p.client == nil || p.client.Exited() {
+		return nil, errors.New("Plugin not running, cannot call resource")
```

Comment:
Errors [should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).
```suggestion
		return nil, errors.New("plugin not running, cannot call resource")
```